src/site/_filters/filter-by-tag.js

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
The main export of this file now is `latestPostByTags`, the file should hence be named `latest-post-by-tags.js`.

Reply:
I have renamed this file from `filter-by-tag.js` to `latest-post-by-tags.js`